tasks/skylab/HISAT2.wdl

```diff
@@ -43,18 +46,33 @@ task HISAT2PairedEnd {
 
     set -e
 
-    # fix names if necessary.
-    if [[ "${fastq1}" != *.fastq.gz ]]; then
-        FQ1=${fastq1}.fastq.gz
-        mv ${fastq1} ${fastq1}.fastq.gz
-    else
-        FQ1=${fastq1}
+   # fix names if necessary
+    if (file ~{fastq1} | grep -q compressed); then
+        if [[ ~{fastq1} != *.gz ]]; then
+            if [[ ~{fastq1} != *.fastq ]]; then
+                FQ1=~{fastq1}.fastq.gz
+                mv  ~{fastq1}  ~{fastq1}.fastq.gz
+            else
+                FQ1=~{fastq1}.gz
+                mv ~{fastq1} ~{fastq1}.gz
+            fi
+        else
+            FQ1=~{fastq1}
+        fi
     fi
```

Comment:
Did a quick google search and it seems the file extension is important for hisat2. We should handle the case where the file. is. not compressed and does not have the `.fastq` or `.fq` suffix

Reply:
Done.